projects/common/lib/components/date/date.component.ts

```diff
@@ -103,7 +115,7 @@ export class DateComponent extends AbstractFormControl
   _defaultErrMsg: ErrorMessage = {
     required: `${LabelReplacementTag} is required.`,
     dayOutOfRange: `Invalid ${LabelReplacementTag}.`,
-    yearDistantPast: `Invalid ${LabelReplacementTag}.`,
+    yearDistantPast: `Invalid ${LabelReplacementTag}`,
```

Comment:
Is this an intentional removal. All the remaining error messages end with a period.